pkg/network/apis/network/types.go

```diff
@@ -22,6 +22,7 @@ type ClusterNetwork struct {
 	HostSubnetLength uint32
 	ServiceNetwork   string
 	PluginName       string
+	VXLANPort        *uint32
```

Comment:
the internal type was missing that field, the field itself was introduced in https://github.com/openshift/api/commit/f7eba8d3fdaee636557e937b7802e2623457a47c (to the external type)

Reply:
@squeed this needs your review. Somehow this got lost in some other change to the network types?

Reply:
@JacobTanenbaum was trying to backport this, IIRC, but we eventually gave up. @knobunc might remember the details.

Reply:
was added in https://github.com/openshift/api/pull/90